_articles/what-is-oauth-understanding-the-authorization-layer.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: article
+title: "What is OAuth? Understanding the authorization layer"
+abstract: "A simple explanation of what OAuth is, how it works and what it is used for."
+sitemap:
+  lastmod: 2020-05-15
+---
+
+OAuth is short for Open Authorization.  OAuth is a framework for token-based authorization on the internet. What does that mean?  Basically, OAuth is a way for websites to share private information with each other. Authorization tokens are issued which grants access to specific pieces of information hence the term token-based authorization.
+
+For example, let’s say you want your social media profile to share with a dating website your name, date of birth and height.  OAuth can be used to authorize the dating app access to those details on your social media profile.  The OAuth framework allows the social media profile to create an access token with certain attributes, such as an expiry date, which the dating website can use to access the data the token has scope for.
```

Comment:
Don't use height that's not a real thing on any website. profile picture instead

Reply:
Also you social media profile isn't the thing doing the sharing its the social media website. 
How about "For example, you want to share your social media profile with a dating website. OAuth can ..."

Reply:
at this level of explination I would avoid using the work authorize too much. 
"OAuth can be used to grant the dating app access to your profile"

Reply:
I find your sentences take quite a lot of digesting because of the number of peices. Is there any loss of clarity from "access the details of your profile" to "access your profile"

Reply:
"The OAuth framework allows the social media profile to create an access token with certain attributes, such as an expiry date, which the dating website can use to access the data the token has scope for."

The previous sentence mentioned OAuth so just "The dating website will receive a secure token that it uses to request the data you shared from (your profile)/ (the social media website)"